apps/meteor/ee/server/api/chat.ts

```diff
@@ -0,0 +1,33 @@
+import { Meteor } from 'meteor/meteor';
+
+import { API } from '../../../app/api/server/api';
+import { hasLicense } from '../../app/license/server/license';
+
+API.v1.addRoute(
+	'chat.getMessageReadReceipts',
+	{ authRequired: true },
+	{
+		async get() {
+			if (!hasLicense('message-read-receipt')) {
+				throw new Meteor.Error('error-action-not-allowed', 'This is an enterprise feature');
+			}
+
+			const { messageId } = this.queryParams;
+			if (!messageId) {
+				return API.v1.failure({
+					error: "The required 'messageId' param is missing.",
+				});
+			}
+
+			try {
+				return API.v1.success({
+					receipts: await Meteor.call('getReadReceipts', { messageId }),
+				});
+			} catch (error: any) {
+				return API.v1.failure({
+					error: error.message,
+				});
+			}
```

Comment:
Since the error is just being returned, do you really need the try/catch or can we rely on the endpoint's try/catch?

Reply:
Hmmm I see we're doing something similar (the try/catch strategy) in some other endpoints. Not sure about what's the ideal behavior here. Maybe @sampaiodiego can help us here

Reply:
this try/catch is redundant.. as @KevLehman mentioned we already have that for every REST endpoint. as you can see [here](https://github.com/RocketChat/Rocket.Chat/blob/2bea8dc006351de264e1424534f0299098c304fd/apps/meteor/app/api/server/api.js#L419)

Reply:
great, fixed!